firestore/testapp/README.md

```diff
@@ -33,6 +33,16 @@ Building and Running the testapp
           sudo gem install cocoapods --pre
         ```
 
+    -   Update the pod versions in the Podfile to match the C++ SDK version that you are using.
+        For the latest version of the C++ SDK, you can find the associated pod versions on the
+        [`Add Firebase to your project` page](https://firebase.google.com/docs/cpp/setup?platform=ios#libraries-ios).
+        For instance: if you're using SDK version 8.2.0, use the following in the Podfile:
```

Comment:
How about adding a clarification like
```
(but note that pod versions may not always match the C++ SDK version)
```
? This is to avoid giving the impression that users can simply use the same version as the C++ SDK because it happens to match in the example.

Reply:
Done